### PR TITLE
FY2019Q3 Quarterly refresh 

### PR DESCRIPTION
**Pull request by andrewconnell**

### CODE REVIEW

File: Demos/01-create-app/graph-tutorial/packages.config [hunk 1]
```diff
@@ -1,50 +1,35 @@
...
+  <package id="jQuery.Validation" version="1.17.0" targetFramework="net472" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net472" />
+  <package id="Microsoft.AspNet.TelemetryCorrelation" version="1.0.5" targetFramework="net472" />
+  <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net472" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net472" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="2.0.1" targetFramework="net472" />
+  <package id="Microsoft.Graph" version="1.12.0" targetFramework="net472" />
+  <package id="Microsoft.Graph.Core" version="1.12.0" targetFramework="net472" />
+  <package id="Microsoft.Identity.Client" version="2.7.1" targetFramework="net472" />
```
Comment (jthake-msft) [inline Demos/01-create-app/graph-tutorial/packages.config]: is there any reason you didn't use 3.0.0-preview?

Comment (andrewconnell) [reply Demos/01-create-app/graph-tutorial/packages.config]: Two reasons: (1) from previous discussions with you, you told us to not upgrade MSAL in any of these until you said to do it & (2) as a general rule for this project with OfficeDev, we have not used preview tech in any modules.

Comment (jthake-msft) [reply Demos/01-create-app/graph-tutorial/packages.config]: OK...just to be clear, all of v2 is -preview, it was a mistake that the -preview tag was removed. This is not your fault you do not know this. For now this is great, we will have to update once the -preview tag is removed from v3.0 around Build time frames.

Comment (andrewconnell) [reply Demos/01-create-app/graph-tutorial/packages.config]: OK... we are planning to update all the modules we're refreshing to v3 & to use the new app registration process in the AAD portal once the preview tags are all removed after Build.

File: Demos/01-create-app/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
\ No newline at end of file
+The version of the project in this directory reflects completing the tutorial up through [Create an ASP.NET MVC web app](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=1). If you use this version of the project, you need to complete the rest of the tutorial starting at [Register the app in the portal](https://docs.microsoft.com/graph/training/aspnet-tutorial?tutorial-step=2).
+
+Installing NuGet packages adds content to the /packages, /Content & /Scripts folders. The files within the /Content & /Scripts folders frequently contain 3rd party libraries which do not include redistributable licenses. Therefore the files in these folders are not included in the repository. The NuGet restore process will not repopulate these folders, only a NuGet reinstall of the package will do that. Therefore you will need to reinstall all packages using the NuGet CLI to repopulate these folders if you want to run the final built solution locally. Refer to the following for more information: [Microsoft Docs – How to reinstall and update packages](https://docs.microsoft.com/en-us/nuget/consume-packages/reinstalling-and-updating-packages).”
```
Comment (jasonjoh) [inline Demos/01-create-app/README.md]: Removing the Content and Scripts folders breaks building these project unless you do the manual steps in PMC to reinstall everything. This is bad from a DevX standpoint. The files in those folder are all distributed under the MIT license which should be fine. Please put these back.

Comment (andrewconnell) [reply Demos/01-create-app/README.md]: I agree... will put these back, however you should be aware of the following.

We are under instructions by OfficeDev to not distribute 3rd party code. In the past security / quality scans of the OfficeDev/TrainingContent repo have identified code comments with swear words & other content MSFT deemed unacceptable. We have to get specific libraries approved by LCA and so far that's only included jQuery & Bootstrap. This was the compromise.

**Pull request merged**